stream_alert/apps/_apps/README.rst

```diff
@@ -7,6 +7,14 @@ To obtain the access key and access key secret, an authorized user of the Aliyun
 
 Additionly, the user for whom the access key was created must have sufficient privileges to make use of ActionTrail; follow the directions on the `Grant ActionTrail permissions to RAM users <https://www.alibabacloud.com/help/doc-detail/28818.htm>`_ page.
 
+How to set up the intercom app
+###########################
+
+The Intercom API requires an access token. Get an access token by following `these instructions <https://developers.intercom.com/building-apps/docs/authorization#section-how-to-get-an-access-token>`.
```

Comment:
The link is not valid RST syntax (you can see this by going to `View File` in the github UI and observing the content rendered improperly in the preview):

![image](https://user-images.githubusercontent.com/13773789/58052818-afc01000-7b0a-11e9-8a70-887bf4d82496.png)

All you need is a trailing `_` after the closing tick (`), like (this won't render because GH only supports markdown):

```
This is a `link <https://www.google.com`_.
```

You can reference some of the other link in this file if need-be :).